signal_scanner_bot/filters.py

```diff
@@ -74,7 +74,11 @@ def _f_retweet_text(status: Status) -> bool:
 
 def _f_not_trusted_tweeter(status: Status) -> bool:
     # Status wasn't sent by a trusted tweeter
-    return status.author.screen_name not in env.TRUSTED_TWEETERS
+    # Do some proper type checking
+    if isinstance(env.TRUSTED_TWEETERS, set):
```

Comment:
I think the original logic was fine...was mypy complaining here? I think the defaults in the original file might need to be modified so that they return empty lists/sets when not defined rather than `""`.

Reply:
After some re-working of the _env function and adding some additional logic and casting I've reverted these changes.